packages/@lwc/features/src/__tests__/flags.spec.ts

```diff
@@ -0,0 +1,307 @@
+              console.log('ENABLE_FEATURE_TRUE');
+            }
+        `,
+    },
+    'should transform boolean-false feature flags': {
+        code: `
+            import { ENABLE_FEATURE_FALSE } from '@lwc/features';
+            if (ENABLE_FEATURE_FALSE) {
+                console.log('ENABLE_FEATURE_FALSE');
+            }
+        `,
+        output: `
+            import { ENABLE_FEATURE_FALSE, runtimeFlags } from '@lwc/features';
+
+            if (runtimeFlags.ENABLE_FEATURE_FALSE) {
+              console.log('ENABLE_FEATURE_FALSE');
+            }
+        `,
+    },
+    'should not tramsform feature flags unless the if-test is a plain indentifier': {
+        code: `
+            import { ENABLE_FEATURE_TRUE, ENABLE_FEATURE_FALSE, ENABLE_FEATURE_NULL } from '@lwc/features';
+            if (ENABLE_FEATURE_NULL === null) {
+                console.log('ENABLE_FEATURE_NULL === null');
+            }
+            if (isTrue(ENABLE_FEATURE_TRUE)) {
+                console.log('isTrue(ENABLE_FEATURE_TRUE)');
+            }
+            if (!ENABLE_FEATURE_FALSE) {
+                console.log('!ENABLE_FEATURE_FALSE');
+            }
+        `,
+        output: `
+            import { ENABLE_FEATURE_TRUE, ENABLE_FEATURE_FALSE, ENABLE_FEATURE_NULL, runtimeFlags } from '@lwc/features';
+
+            if (ENABLE_FEATURE_NULL === null) {
+              console.log('ENABLE_FEATURE_NULL === null');
+            }
+
+            if (isTrue(ENABLE_FEATURE_TRUE)) {
+              console.log('isTrue(ENABLE_FEATURE_TRUE)');
+            }
+
+            if (!ENABLE_FEATURE_FALSE) {
+              console.log('!ENABLE_FEATURE_FALSE');
+            }
+        `,
+    },
+    'should not transform identifiers that look like feature flags but are not imported': {
+        code: `
+            import { ENABLE_FEATURE_TRUE } from '@lwc/features';
+            const ENABLE_FEATURE_FALSE = true;
+            if (ENABLE_FEATURE_FALSE) {
+                console.log('ENABLE_FEATURE_FALSE');
+            }
+            function awesome() {
+                const ENABLE_FEATURE_TRUE = null;
+                if (ENABLE_FEATURE_TRUE) {
+                    console.log('ENABLE_FEATURE_TRUE');
+                }
+            }
+        `,
+        output: `
+            import { ENABLE_FEATURE_TRUE, runtimeFlags } from '@lwc/features';
+            const ENABLE_FEATURE_FALSE = true;
+
+            if (ENABLE_FEATURE_FALSE) {
+              console.log('ENABLE_FEATURE_FALSE');
+            }
+
+            function awesome() {
+              const ENABLE_FEATURE_TRUE = null;
+
+              if (ENABLE_FEATURE_TRUE) {
+                console.log('ENABLE_FEATURE_TRUE');
+              }
+            }
+        `,
+    },
+    'should not transform feature flags when used with a ternary operator': {
+        code: `
+            import { ENABLE_FEATURE_NULL } from '@lwc/features';
+            console.log(ENABLE_FEATURE_NULL ? 'foo' : 'bar');
```

Comment:
why not?

Reply:
I went with a simple implementation where feature flags only work for if-statements since I couldn't think of a scenario where that wouldn't be sufficient. I can transform feature flags without any restrictions if we think that's useful.